doc/nrf/ug_thingy53.rst

```diff
@@ -260,60 +187,158 @@ You can program the precompiled firmware image using one of the following ways:
+   To use your application with the Nordic Thingy:53 preprogrammed bootloader, you must only set the proper configuration for the Partition Manager and MCUboot.
+   Configurations that enable the other features are not mandatory, but you can enable them if needed.
+
+   All applications compatible with the Nordic Thingy:53 in the |NCS| also support the additional features.
+   Refer to the source code of the samples and applications for examples of implementation.
+
+.. _thingy53_app_partition_manager_config:
 
 Partition manager configuration
-  Thingy:53 compatible samples use :ref:`partition_manager` to define memory partitions.
-  To ensure that partition layout does not change between builds, the sample must use static partition layout that is the same as the one in Thingy:53-compatible samples in the |NCS|.
-  The memory layout must stay consistent, so that MCUboot can perform proper image updates and clean up the settings storage partition.
+===============================
 
-  The PCD SRAM partition is locked by the MCUboot bootloader to prevent the application from modifying the network core firmware.
-  Trying to access data on this partition results in an ARM fault.
+The samples compatible with the Nordic Thingy:53 use :ref:`partition_manager` to define memory partitions.
+To ensure that partition layout does not change between builds, the sample must use a static partition layout that is the same for all compatible samples in the |NCS|.
+See :file:`pm_static_thingy53_nrf5340_cpuapp.yml` in a compatible sample for the configuration used.
+The memory layout must stay consistent, so that MCUboot can perform proper image updates and clean up the settings storage partition.
+
+The PCD SRAM partition is locked by the MCUboot bootloader to prevent the application from modifying the network core firmware.
+Trying to access data on this partition results in an ARM fault.
+
+It is also necessary to add a flash controller overlay for the application core to allow data to be loaded from the external flash memory to the network core using the application core RAM.
+This is done because the network core cannot access the external flash memory directly.
+See :file:`thingy53_nrf5340_cpuapp.overlay` in a compatible sample for how this is done.
+
+.. _thingy53_app_mcuboot_bootloader:
 
 MCUboot bootloader
-  The sample must enable MCUboot bootloader and use the same configuration of the bootloader as all of the Thingy:53-compatible samples in the |NCS|.
-  This is needed to let MCUboot use the sample as an image update.
+==================
 
-  The MCUboot bootloader supports serial recovery and a custom command to erase settings storage partition.
-  Erasing settings parition is needed to ensure that an application would not be booted with incompatible content of settings partition.
+Each sample must enable the MCUboot bootloader and use the same configuration of the bootloader as all of the Nordic Thingy:53-compatible samples in the |NCS|.
```

Comment:
We could mention that MCUboot configuration for Thingy:53 is added as Thingy:53's board specific configuration overlay defined in the MCUboot repository (to help users locate it quickly)

Reply:
There's an overlay file that users need to get from the repository in addition to using the Kconfig options presented in this section? Or there is an overlay file that can be used to set these instead of doing it manually?

Reply:
The MCUboot repository contains default configuration for Thingy:53 which is taken into account when building MCUboot bootloader for Thingy:53 in the NCS (for both secure and not secure variant). The configuration is added automatically when MCUboot is built for Thingy:53, user does not to do anything manually.

Reply:
Would adding the following sentence as a third sentence in the first paragraph convey this:
`When using |NCS| to build the MCUboot bootloader for the Thingy:53, the configuration is applied automatically from the MCUboot repository.`